package.json

```diff
@@ -39,13 +39,13 @@
   "devDependencies": {
     "@newrelic/eslint-config": "^0.0.4",
     "@newrelic/newrelic-oss-cli": "^0.1.2",
-    "@newrelic/test-utilities": "^6.4.0",
-    "husky": "^7.0.4",
-    "lint-staged": "^12.3.3",
-    "newrelic": "^8.8.0",
-    "tap": "^16.0.1"
+    "@newrelic/test-utilities": "6.5.3",
```

Comment:
we don't like to pin versions.  the best approach here would be to `npm i <pkg>@latest` which will upgrade it to the latest but keep the `^` matching